src/components/molecules/NavbarSchedule/NavbarSchedule.scss

```diff
@@ -0,0 +1,48 @@
+@import "scss/constants.scss";
+
+.nav-party-logo {
+  position: relative;
+  background-color: rgba($white, 0.08);
+  height: 60px;
```

Comment:
Can we use variables for this + similar locations within here. Particularly when heights/etc are 'calculated', we should show the actual calculations rather than just using 'precomputed magic numbers'.

---

Specifically with this `height`, do we need to define it here? Can we use `100%` or similar?

It looks like the `NavBar` already defines it's height specifically (`src/components/molecules/NavBar/playa.scss:25`), and then this class is rendered within a `div` inside it. So I believe it will inherit the same height/etc.

Reply:
Yup, that's right. This height is unnecessary.